

==================[30.07.2019 19:26:14]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[30.07.2019 19:26:14]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[30.07.2019 19:45:58]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[30.07.2019 19:45:58]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[30.07.2019 19:59:25]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[30.07.2019 19:59:25]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[30.07.2019 20:27:12]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[30.07.2019 20:27:13]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[30.07.2019 20:36:59]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[30.07.2019 20:36:59]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[30.07.2019 23:56:31]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[30.07.2019 23:56:31]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[30.07.2019 23:56:33]==================
System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
=========================================================

==================[30.07.2019 23:56:33]==================
System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
=========================================================

==================[30.07.2019 23:56:33]==================
System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
=========================================================

==================[30.07.2019 23:56:33]==================
System.AggregateException: One or more errors occurred. (Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306) ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException: Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   --- End of inner exception stack trace ---
   at bool System.Threading.Tasks.Task.Wait(int millisecondsTimeout, CancellationToken cancellationToken)
---> (Inner Exception #0) System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)<---

=========================================================

==================[30.07.2019 23:56:33]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'MySql.Data.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[30.07.2019 23:56:33]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'MySql.Data.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[30.07.2019 23:56:33]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts. ---> System.AggregateException: One or more errors occurred. (Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306) ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException: Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   --- End of inner exception stack trace ---
   at bool System.Threading.Tasks.Task.Wait(int millisecondsTimeout, CancellationToken cancellationToken)
   at Stream MySql.Data.Common.StreamCreator.GetTcpStream(MySqlConnectionStringBuilder settings)
   at void MySql.Data.MySqlClient.NativeDriver.Open()
---> (Inner Exception #0) System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)<---

   at void MySql.Data.MySqlClient.NativeDriver.Open()
=========================================================

==================[30.07.2019 23:56:34]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts. ---> System.AggregateException: One or more errors occurred. (Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306) ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException: Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   --- End of inner exception stack trace ---
   at bool System.Threading.Tasks.Task.Wait(int millisecondsTimeout, CancellationToken cancellationToken)
   at Stream MySql.Data.Common.StreamCreator.GetTcpStream(MySqlConnectionStringBuilder settings)
   at void MySql.Data.MySqlClient.NativeDriver.Open()
---> (Inner Exception #0) System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)<---

   at void MySql.Data.MySqlClient.NativeDriver.Open()
   at void MySql.Data.MySqlClient.Driver.Open()
   at Driver MySql.Data.MySqlClient.Driver.Create(MySqlConnectionStringBuilder settings)
=========================================================

==================[30.07.2019 23:56:34]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts. ---> System.AggregateException: One or more errors occurred. (Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306) ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException: Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   --- End of inner exception stack trace ---
   at bool System.Threading.Tasks.Task.Wait(int millisecondsTimeout, CancellationToken cancellationToken)
   at Stream MySql.Data.Common.StreamCreator.GetTcpStream(MySqlConnectionStringBuilder settings)
   at void MySql.Data.MySqlClient.NativeDriver.Open()
---> (Inner Exception #0) System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)<---

   at void MySql.Data.MySqlClient.NativeDriver.Open()
   at void MySql.Data.MySqlClient.Driver.Open()
   at Driver MySql.Data.MySqlClient.Driver.Create(MySqlConnectionStringBuilder settings)
   at Driver MySql.Data.MySqlClient.MySqlPool.CreateNewPooledConnection()
   at Driver MySql.Data.MySqlClient.MySqlPool.GetPooledConnection()
   at Driver MySql.Data.MySqlClient.MySqlPool.TryToGetDriver()
=========================================================

==================[30.07.2019 23:56:34]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts. ---> System.AggregateException: One or more errors occurred. (Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306) ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException: Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   --- End of inner exception stack trace ---
   at bool System.Threading.Tasks.Task.Wait(int millisecondsTimeout, CancellationToken cancellationToken)
   at Stream MySql.Data.Common.StreamCreator.GetTcpStream(MySqlConnectionStringBuilder settings)
   at void MySql.Data.MySqlClient.NativeDriver.Open()
---> (Inner Exception #0) System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)<---

   at void MySql.Data.MySqlClient.NativeDriver.Open()
   at void MySql.Data.MySqlClient.Driver.Open()
   at Driver MySql.Data.MySqlClient.Driver.Create(MySqlConnectionStringBuilder settings)
   at Driver MySql.Data.MySqlClient.MySqlPool.CreateNewPooledConnection()
   at Driver MySql.Data.MySqlClient.MySqlPool.GetPooledConnection()
   at Driver MySql.Data.MySqlClient.MySqlPool.TryToGetDriver()
   at Driver MySql.Data.MySqlClient.MySqlPool.GetConnection()
   at void MySql.Data.MySqlClient.MySqlConnection.Open()
=========================================================

==================[30.07.2019 23:57:45]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[30.07.2019 23:57:45]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[01.08.2019 20:10:31]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[01.08.2019 20:10:32]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[01.08.2019 20:17:26]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[01.08.2019 20:17:26]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[01.08.2019 20:53:48]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[01.08.2019 20:53:48]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[01.08.2019 20:54:51]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'Microsoft.CSharp.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[01.08.2019 20:54:51]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'Microsoft.CSharp.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[01.08.2019 20:54:52]==================
Microsoft.CSharp.RuntimeBinder.RuntimeBinderException: 'Newtonsoft.Json.Linq.JArray' does not contain a definition for 'Length'
   at Expr Microsoft.CSharp.RuntimeBinder.RuntimeBinder.BindProperty(ICSharpBinder payload, ArgumentObject argument, LocalVariableSymbol local, Expr optionalIndexerArguments)
=========================================================

==================[01.08.2019 20:54:52]==================
System.NotImplementedException: The method or operation is not implemented.
   at IList<CustomAttributeData> System.Reflection.MemberInfo.GetCustomAttributesData()
=========================================================

==================[01.08.2019 20:54:52]==================
Microsoft.CSharp.RuntimeBinder.RuntimeBinderException: 'Newtonsoft.Json.Linq.JArray' does not contain a definition for 'Length'
   at object CallSite.Target(Closure, CallSite, object)
=========================================================

==================[01.08.2019 20:54:52]==================
System.NotImplementedException: The method or operation is not implemented.
   at IList<CustomAttributeData> System.Reflection.MemberInfo.GetCustomAttributesData()
=========================================================

==================[01.08.2019 20:54:52]==================
System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> Microsoft.CSharp.RuntimeBinder.RuntimeBinderException: 'Newtonsoft.Json.Linq.JArray' does not contain a definition for 'Length'
   at object CallSite.Target(Closure, CallSite, object)
   at TRet System.Dynamic.UpdateDelegates.UpdateAndExecute1<T0, TRet>(CallSite site, T0 arg0)
   at void Wave.Character.Customization.OnCloseCustomization(Client player, string playerJsonInfo, string name, string surname, int gender, int age, string clothes) in C:\Users\Oniel\Documents\GitHub\WaveRP\bridge\resources\Wave\Character\Customization.cs:line 21
   --- End of inner exception stack trace ---
   at object RuntimeMethodHandle.InvokeMethod(object target, object[] arguments, Signature sig, bool constructor, bool wrapExceptions)
   at object System.Reflection.RuntimeMethodInfo.Invoke(object obj, BindingFlags invokeAttr, Binder binder, object[] parameters, CultureInfo culture)
=========================================================

==================[01.08.2019 20:54:52]==================
System.NotImplementedException: The method or operation is not implemented.
   at IList<CustomAttributeData> System.Reflection.MemberInfo.GetCustomAttributesData()
=========================================================

==================[01.08.2019 21:01:42]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[01.08.2019 21:01:42]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[01.08.2019 21:26:20]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[01.08.2019 21:26:20]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[01.08.2019 21:27:20]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'MySql.Data.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[01.08.2019 21:27:20]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'MySql.Data.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[01.08.2019 21:27:20]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Parameter '@characterid' has already been defined.
   at MySqlParameter MySql.Data.MySqlClient.MySqlParameterCollection.InternalAdd(MySqlParameter value, int index)
=========================================================

==================[01.08.2019 21:37:04]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[01.08.2019 21:37:04]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[01.08.2019 21:45:23]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[01.08.2019 21:45:23]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[01.08.2019 22:04:02]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[01.08.2019 22:04:02]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[01.08.2019 22:04:04]==================
System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
=========================================================

==================[01.08.2019 22:04:04]==================
System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
=========================================================

==================[01.08.2019 22:04:04]==================
System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
=========================================================

==================[01.08.2019 22:04:05]==================
System.AggregateException: One or more errors occurred. (Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306) ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException: Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   --- End of inner exception stack trace ---
   at bool System.Threading.Tasks.Task.Wait(int millisecondsTimeout, CancellationToken cancellationToken)
---> (Inner Exception #0) System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)<---

=========================================================

==================[01.08.2019 22:04:05]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'MySql.Data.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[01.08.2019 22:04:05]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'MySql.Data.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[01.08.2019 22:04:05]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts. ---> System.AggregateException: One or more errors occurred. (Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306) ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException: Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   --- End of inner exception stack trace ---
   at bool System.Threading.Tasks.Task.Wait(int millisecondsTimeout, CancellationToken cancellationToken)
   at Stream MySql.Data.Common.StreamCreator.GetTcpStream(MySqlConnectionStringBuilder settings)
   at void MySql.Data.MySqlClient.NativeDriver.Open()
---> (Inner Exception #0) System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)<---

   at void MySql.Data.MySqlClient.NativeDriver.Open()
=========================================================

==================[01.08.2019 22:04:05]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts. ---> System.AggregateException: One or more errors occurred. (Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306) ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException: Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   --- End of inner exception stack trace ---
   at bool System.Threading.Tasks.Task.Wait(int millisecondsTimeout, CancellationToken cancellationToken)
   at Stream MySql.Data.Common.StreamCreator.GetTcpStream(MySqlConnectionStringBuilder settings)
   at void MySql.Data.MySqlClient.NativeDriver.Open()
---> (Inner Exception #0) System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)<---

   at void MySql.Data.MySqlClient.NativeDriver.Open()
   at void MySql.Data.MySqlClient.Driver.Open()
   at Driver MySql.Data.MySqlClient.Driver.Create(MySqlConnectionStringBuilder settings)
=========================================================

==================[01.08.2019 22:04:05]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts. ---> System.AggregateException: One or more errors occurred. (Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306) ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException: Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   --- End of inner exception stack trace ---
   at bool System.Threading.Tasks.Task.Wait(int millisecondsTimeout, CancellationToken cancellationToken)
   at Stream MySql.Data.Common.StreamCreator.GetTcpStream(MySqlConnectionStringBuilder settings)
   at void MySql.Data.MySqlClient.NativeDriver.Open()
---> (Inner Exception #0) System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)<---

   at void MySql.Data.MySqlClient.NativeDriver.Open()
   at void MySql.Data.MySqlClient.Driver.Open()
   at Driver MySql.Data.MySqlClient.Driver.Create(MySqlConnectionStringBuilder settings)
   at Driver MySql.Data.MySqlClient.MySqlPool.CreateNewPooledConnection()
   at Driver MySql.Data.MySqlClient.MySqlPool.GetPooledConnection()
   at Driver MySql.Data.MySqlClient.MySqlPool.TryToGetDriver()
=========================================================

==================[01.08.2019 22:04:05]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts. ---> System.AggregateException: One or more errors occurred. (Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306) ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException: Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   --- End of inner exception stack trace ---
   at bool System.Threading.Tasks.Task.Wait(int millisecondsTimeout, CancellationToken cancellationToken)
   at Stream MySql.Data.Common.StreamCreator.GetTcpStream(MySqlConnectionStringBuilder settings)
   at void MySql.Data.MySqlClient.NativeDriver.Open()
---> (Inner Exception #0) System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)<---

   at void MySql.Data.MySqlClient.NativeDriver.Open()
   at void MySql.Data.MySqlClient.Driver.Open()
   at Driver MySql.Data.MySqlClient.Driver.Create(MySqlConnectionStringBuilder settings)
   at Driver MySql.Data.MySqlClient.MySqlPool.CreateNewPooledConnection()
   at Driver MySql.Data.MySqlClient.MySqlPool.GetPooledConnection()
   at Driver MySql.Data.MySqlClient.MySqlPool.TryToGetDriver()
   at Driver MySql.Data.MySqlClient.MySqlPool.GetConnection()
   at void MySql.Data.MySqlClient.MySqlConnection.Open()
=========================================================

==================[01.08.2019 22:05:00]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[01.08.2019 22:05:00]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[01.08.2019 22:28:32]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[01.08.2019 22:28:32]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[01.08.2019 22:44:26]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[01.08.2019 22:44:26]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[01.08.2019 22:50:27]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[01.08.2019 22:50:27]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[01.08.2019 23:01:11]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[01.08.2019 23:01:11]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[01.08.2019 23:23:05]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[01.08.2019 23:23:05]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[01.08.2019 23:32:59]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[01.08.2019 23:32:59]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[01.08.2019 23:38:16]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[01.08.2019 23:38:16]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[02.08.2019 1:21:16]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[02.08.2019 1:21:16]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[02.08.2019 17:52:46]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[02.08.2019 17:52:46]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[02.08.2019 22:42:24]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[02.08.2019 22:42:24]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[02.08.2019 22:49:33]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[02.08.2019 22:49:33]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[02.08.2019 23:17:32]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[02.08.2019 23:17:32]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[03.08.2019 22:24:16]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[03.08.2019 22:24:17]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[03.08.2019 22:24:19]==================
System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
=========================================================

==================[03.08.2019 22:24:19]==================
System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
=========================================================

==================[03.08.2019 22:24:19]==================
System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)
   at void System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
=========================================================

==================[03.08.2019 22:24:19]==================
System.AggregateException: One or more errors occurred. (Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306) ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException: Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   --- End of inner exception stack trace ---
   at bool System.Threading.Tasks.Task.Wait(int millisecondsTimeout, CancellationToken cancellationToken)
---> (Inner Exception #0) System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)<---

=========================================================

==================[03.08.2019 22:24:19]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'MySql.Data.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[03.08.2019 22:24:19]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'MySql.Data.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[03.08.2019 22:24:20]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts. ---> System.AggregateException: One or more errors occurred. (Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306) ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException: Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   --- End of inner exception stack trace ---
   at bool System.Threading.Tasks.Task.Wait(int millisecondsTimeout, CancellationToken cancellationToken)
   at Stream MySql.Data.Common.StreamCreator.GetTcpStream(MySqlConnectionStringBuilder settings)
   at void MySql.Data.MySqlClient.NativeDriver.Open()
---> (Inner Exception #0) System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)<---

   at void MySql.Data.MySqlClient.NativeDriver.Open()
=========================================================

==================[03.08.2019 22:24:20]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts. ---> System.AggregateException: One or more errors occurred. (Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306) ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException: Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   --- End of inner exception stack trace ---
   at bool System.Threading.Tasks.Task.Wait(int millisecondsTimeout, CancellationToken cancellationToken)
   at Stream MySql.Data.Common.StreamCreator.GetTcpStream(MySqlConnectionStringBuilder settings)
   at void MySql.Data.MySqlClient.NativeDriver.Open()
---> (Inner Exception #0) System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)<---

   at void MySql.Data.MySqlClient.NativeDriver.Open()
   at void MySql.Data.MySqlClient.Driver.Open()
   at Driver MySql.Data.MySqlClient.Driver.Create(MySqlConnectionStringBuilder settings)
=========================================================

==================[03.08.2019 22:24:20]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts. ---> System.AggregateException: One or more errors occurred. (Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306) ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException: Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   --- End of inner exception stack trace ---
   at bool System.Threading.Tasks.Task.Wait(int millisecondsTimeout, CancellationToken cancellationToken)
   at Stream MySql.Data.Common.StreamCreator.GetTcpStream(MySqlConnectionStringBuilder settings)
   at void MySql.Data.MySqlClient.NativeDriver.Open()
---> (Inner Exception #0) System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)<---

   at void MySql.Data.MySqlClient.NativeDriver.Open()
   at void MySql.Data.MySqlClient.Driver.Open()
   at Driver MySql.Data.MySqlClient.Driver.Create(MySqlConnectionStringBuilder settings)
   at Driver MySql.Data.MySqlClient.MySqlPool.CreateNewPooledConnection()
   at Driver MySql.Data.MySqlClient.MySqlPool.GetPooledConnection()
   at Driver MySql.Data.MySqlClient.MySqlPool.TryToGetDriver()
=========================================================

==================[03.08.2019 22:24:20]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts. ---> System.AggregateException: One or more errors occurred. (Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306) ---> System.Net.Internals.SocketExceptionFactory+ExtendedSocketException: Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)
   --- End of inner exception stack trace ---
   at bool System.Threading.Tasks.Task.Wait(int millisecondsTimeout, CancellationToken cancellationToken)
   at Stream MySql.Data.Common.StreamCreator.GetTcpStream(MySqlConnectionStringBuilder settings)
   at void MySql.Data.MySqlClient.NativeDriver.Open()
---> (Inner Exception #0) System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (10061): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at bool System.Net.Sockets.Socket.DoMultipleAddressConnectCallback(object result, MultipleAddressConnectAsyncResult context)
   at void System.Net.Sockets.Socket.MultipleAddressConnectCallback(IAsyncResult result)
   at void System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at void System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   at void System.Threading.Tasks.TaskFactory<TResult>.FromAsyncCoreLogic(IAsyncResult iar, Func<IAsyncResult, TResult> endFunction, Action<IAsyncResult> endAction, Task<TResult> promise, bool requiresSynchronization)<---

   at void MySql.Data.MySqlClient.NativeDriver.Open()
   at void MySql.Data.MySqlClient.Driver.Open()
   at Driver MySql.Data.MySqlClient.Driver.Create(MySqlConnectionStringBuilder settings)
   at Driver MySql.Data.MySqlClient.MySqlPool.CreateNewPooledConnection()
   at Driver MySql.Data.MySqlClient.MySqlPool.GetPooledConnection()
   at Driver MySql.Data.MySqlClient.MySqlPool.TryToGetDriver()
   at Driver MySql.Data.MySqlClient.MySqlPool.GetConnection()
   at void MySql.Data.MySqlClient.MySqlConnection.Open()
=========================================================

==================[03.08.2019 22:35:58]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[03.08.2019 22:35:58]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[03.08.2019 22:39:43]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[03.08.2019 22:39:43]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[06.08.2019 17:44:13]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[06.08.2019 17:44:13]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[07.08.2019 16:08:20]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[07.08.2019 16:08:20]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[07.08.2019 16:08:28]==================
System.OutOfMemoryException: Exception of type 'System.OutOfMemoryException' was thrown.
   at void Wave.Account.Login.OnCharSelected(Client player, string characterName)+() => { } in C:\Users\Oniel\Documents\GitHub\WaveRP\bridge\resources\Wave\Account\Login.cs:line 157
   at void System.Collections.Generic.List<T>.ForEach(Action<T> action)
=========================================================

==================[07.08.2019 16:40:13]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[07.08.2019 16:40:13]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[09.08.2019 18:33:42]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[09.08.2019 18:33:42]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[09.08.2019 18:36:22]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[09.08.2019 18:36:22]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[09.08.2019 22:44:31]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[09.08.2019 22:44:31]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[09.08.2019 23:03:03]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[09.08.2019 23:03:03]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[10.08.2019 20:13:21]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[10.08.2019 20:13:21]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[12.08.2019 19:39:26]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[12.08.2019 19:39:26]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[12.08.2019 19:43:29]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[12.08.2019 19:43:29]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[12.08.2019 19:50:25]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[12.08.2019 19:50:25]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[12.08.2019 19:53:11]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[12.08.2019 19:53:11]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[12.08.2019 20:02:56]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[12.08.2019 20:02:56]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[13.08.2019 14:46:47]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[13.08.2019 14:46:47]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[13.08.2019 14:51:19]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[13.08.2019 14:51:19]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[13.08.2019 20:16:37]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[13.08.2019 20:16:37]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[13.08.2019 20:19:15]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[13.08.2019 20:19:15]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[13.08.2019 20:40:25]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[13.08.2019 20:40:25]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[14.08.2019 15:36:16]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[14.08.2019 15:36:16]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[14.08.2019 15:55:51]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[14.08.2019 15:55:51]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[14.08.2019 15:59:59]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[14.08.2019 15:59:59]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[14.08.2019 16:07:51]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[14.08.2019 16:07:51]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[14.08.2019 16:16:53]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[14.08.2019 16:16:53]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[14.08.2019 16:43:59]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[14.08.2019 16:43:59]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[14.08.2019 17:24:45]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[14.08.2019 17:24:45]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[14.08.2019 17:49:48]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[14.08.2019 17:49:48]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[14.08.2019 18:03:02]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[14.08.2019 18:03:02]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[14.08.2019 23:15:59]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[14.08.2019 23:15:59]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[14.08.2019 23:32:54]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[14.08.2019 23:32:54]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[15.08.2019 16:37:10]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[15.08.2019 16:37:10]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[15.08.2019 16:43:15]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[15.08.2019 16:43:15]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[15.08.2019 16:51:10]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[15.08.2019 16:51:10]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[15.08.2019 16:51:42]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '1' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
=========================================================

==================[15.08.2019 16:51:43]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '1' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at int MySql.Data.MySqlClient.NativeDriver.GetResult(ref int affectedRow, ref long insertedId)
=========================================================

==================[15.08.2019 16:51:43]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '1' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at int MySql.Data.MySqlClient.NativeDriver.GetResult(ref int affectedRow, ref long insertedId)
   at ResultSet MySql.Data.MySqlClient.Driver.NextResult(int statementId, bool force)
   at bool MySql.Data.MySqlClient.MySqlDataReader.NextResult()
=========================================================

==================[15.08.2019 16:51:43]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '1' for key 'PRIMARY'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at int MySql.Data.MySqlClient.NativeDriver.GetResult(ref int affectedRow, ref long insertedId)
   at ResultSet MySql.Data.MySqlClient.Driver.NextResult(int statementId, bool force)
   at bool MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   at MySqlDataReader MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
=========================================================

==================[15.08.2019 17:13:53]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[15.08.2019 17:13:53]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[15.08.2019 17:20:49]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[15.08.2019 17:20:49]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[15.08.2019 17:25:23]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[15.08.2019 17:25:23]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[15.08.2019 17:53:34]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[15.08.2019 17:53:34]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[15.08.2019 19:31:25]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[15.08.2019 19:31:25]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[15.08.2019 19:41:36]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[15.08.2019 19:41:36]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[15.08.2019 19:45:12]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[15.08.2019 19:45:12]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[15.08.2019 20:07:47]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[15.08.2019 20:07:47]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[15.08.2019 20:14:02]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[15.08.2019 20:14:02]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[25.08.2019 21:30:42]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[25.08.2019 21:30:42]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[25.08.2019 21:44:14]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[25.08.2019 21:44:14]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[26.08.2019 14:04:14]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[26.08.2019 14:04:14]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[27.08.2019 19:35:08]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[27.08.2019 19:35:08]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[27.08.2019 19:48:00]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[27.08.2019 19:48:00]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[27.08.2019 20:13:36]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[27.08.2019 20:13:36]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[27.08.2019 20:21:21]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[27.08.2019 20:21:21]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[27.08.2019 20:43:42]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[27.08.2019 20:43:42]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[27.08.2019 20:46:24]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[27.08.2019 20:46:24]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[27.08.2019 21:03:48]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[27.08.2019 21:03:48]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[27.08.2019 21:11:36]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[27.08.2019 21:11:37]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[27.08.2019 21:28:13]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[27.08.2019 21:28:13]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[27.08.2019 21:35:15]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[27.08.2019 21:35:15]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[30.08.2019 11:10:20]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[30.08.2019 11:10:20]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[30.08.2019 11:11:55]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'MySql.Data.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[30.08.2019 11:11:55]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'MySql.Data.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[30.08.2019 11:11:55]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Parameter '@nickName' has already been defined.
   at MySqlParameter MySql.Data.MySqlClient.MySqlParameterCollection.InternalAdd(MySqlParameter value, int index)
=========================================================

==================[30.08.2019 11:20:17]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[30.08.2019 11:20:17]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[30.08.2019 11:31:59]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[30.08.2019 11:31:59]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[30.08.2019 21:30:15]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[30.08.2019 21:30:15]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[22.09.2019 18:29:57]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[22.09.2019 18:29:57]==================
System.IO.FileNotFoundException: Could not load the specified file.
File name: 'System.Net.Sockets.resources'
   at Assembly System.Runtime.Loader.AssemblyLoadContext.ResolveUsingEvent(AssemblyName assemblyName)
=========================================================

==================[22.09.2019 18:29:57]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Unknown column 'login' in 'where clause'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
=========================================================

==================[22.09.2019 18:29:57]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Unknown column 'login' in 'where clause'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at int MySql.Data.MySqlClient.NativeDriver.GetResult(ref int affectedRow, ref long insertedId)
=========================================================

==================[22.09.2019 18:29:57]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Unknown column 'login' in 'where clause'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at int MySql.Data.MySqlClient.NativeDriver.GetResult(ref int affectedRow, ref long insertedId)
   at ResultSet MySql.Data.MySqlClient.Driver.NextResult(int statementId, bool force)
   at bool MySql.Data.MySqlClient.MySqlDataReader.NextResult()
=========================================================

==================[22.09.2019 18:29:57]==================
MySql.Data.MySqlClient.MySqlException (0x80004005): Unknown column 'login' in 'where clause'
   at MySqlPacket MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   at int MySql.Data.MySqlClient.NativeDriver.GetResult(ref int affectedRow, ref long insertedId)
   at ResultSet MySql.Data.MySqlClient.Driver.NextResult(int statementId, bool force)
   at bool MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   at MySqlDataReader MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
=========================================================
